src/java.base/share/classes/jdk/internal/misc/Signal.java

```diff
@@ -102,11 +102,10 @@ public boolean equals(Object other) {
         if (this == other) {
```

Comment:
It might be a bit cleaner to rename Object other to "obj" to avoid having Object other and Signal other1.

Reply:
Actually, I'm not sure if `oth` is better name for variable than `other1`.
I would say they have the same rank :)

Reply:
I believe Alan is suggesting to do:

```
    /**
     * Compares the equality of two <code>Signal</code> objects.
     *
     * @param obj the object to compare with.
     * @return whether two <code>Signal</code> objects are equal.
     */
    public boolean equals(Object obj) {
        if (this == obj) {
```

this leaves the variable name `other` free for later use inside the method.

Reply:
> Actually, I'm not sure if `oth` is better name for variable than `other1`.
> I would say they have the same rank :)

Sorry, I should have been clearer, the comment was about equals(Object other). If you rename "other" then it will avoid "other1"